src/mars.c

```diff
@@ -1808,6 +1826,27 @@ static const char* parse_arch(size_t argc, const char** argv, const char* arch)
     return arch;
 }
 
+/***********************************
+ * Parse command line arguments for -conf=path.
+ */
+
+static const char* parse_conf_arg(size_t argc, const char** argv)
+{
+    const char *conf=NULL;
+    for (size_t i = 0; i < argc; ++i)
+    {
+        const char* p = argv[i];
+        if (p[0] == '-')
+        {
+            if (strncmp(p + 1, "conf=", 5) == 0)
+                conf = p + 6;
+            else if (strcmp(p + 1, "run") == 0)
```

Comment:
In `dmd -run myprog.d -conf=dmd.conf` the `-conf=dmd.conf` argument belongs to myprog.